includes/class-content-toc.php

```diff
@@ -134,10 +135,12 @@ public function shortcode( $atts, $content = null ) {
 
 			$toc_html .= apply_filters(
 				'hm_content_toc_list',
-				'<ul id="hm-content-toc-' . esc_attr( $this->id_counter ) . '" class="hm-content-toc">' . $toc_list . '</ul>',
+				'<ul class="hm-content-toc">' . $toc_list . '</ul>',
```

Comment:
I'm not sure how useful a filter on HTML itself is - you can leave this one in place as it'll still do the same thing.

You could have a filter on the title itself, even the entire output, but I prefer to have filters on things like arrays / other variables used to generate the markup in the first place.